Source/JavaScriptCore/runtime/IntlDisplayNames.h

```diff
@@ -29,6 +29,13 @@
 #include <unicode/uldnames.h>
 #include <wtf/unicode/icu/ICUHelpers.h>
 
+#if !defined(HAVE_ICU_U_LOCALE_DISPLAY_NAMES)
+// We need 61 or later since part of implementation uses UCURR_NARROW_SYMBOL_NAME.
+#if U_ICU_VERSION_MAJOR_NUM >= 61
+#define HAVE_ICU_U_LOCALE_DISPLAY_NAMES 1
+#endif
+#endif
```

Comment:
Can we put this into `PlatformHave.h` instead? One advantage of that is that we won’t have to add includes. But another is that’s how we intend to deal with `HAVE()` whenever possible.

Reply:
I think we cannot do that since they are relying on ICU headers, and Platform.h cannot include C code because it is used for sandbox processing too.

Reply:
Yeah we really don't want Platform.h to depend on ICU headers.